front/pages/w/[wId]/join.tsx

```diff
@@ -115,62 +115,74 @@ export default function Join({
-            <P>
-              Please log in or sign up with your company email to access this
-              conversation.
-            </P>
-          ) : (
-            <P>
-              You've been invited to join the{" "}
-              <Strong>{workspace.name} workspace on Dust</Strong>.
-            </P>
-          )}
-        </div>
+    <OnboardingLayout
+      owner={workspace}
+      gaTrackingId={gaTrackingId}
+      headerTitle="Welcome to Dust"
+      headerRightActions={
+        <SignInButton
+          label="Sign up with Google"
+          icon={GoogleLogo}
+          size="sm"
+          onClick={handleSignUpClick}
+        />
+      }
+    >
+      <div className="flex flex-col gap-8">
+        <Page.Header
+          title={`Hello there!`}
+          icon={() => <Logo className="-ml-8 h-4 w-32" />}
+        />
+        <div className="flex flex-col gap-4">
+          <div className="flex flex-col gap-1">
+            <p>Welcome aboard!</p>
+            {onboardingType === "domain_conversation_link" ? (
+              <p>
+                Please log in or sign up with your company email to access this
+                conversation.
+              </p>
+            ) : (
+              <p>
+                You've been invited to join{" "}
+                <strong>{workspace.name}'s workspace on Dust</strong>.
+              </p>
+            )}
+          </div>
 
-        {onboardingType === "email_invite" && (
-          <P>How would you like to connect?</P>
-        )}
+          <p>
+            Dust is a platform giving you access to the best AI assistants. It's
+            easy to use and it's a great place for teams to collaborate. Learn
+            more about Dust on{" "}
+            <A href="https://dust.tt" target="_blank">
+              our website
+            </A>
+            .
+          </p>
+        </div>
 
         <div className="flex flex-col items-center justify-center gap-4 ">
-          <SignInButton
-            label="Sign up with Google"
-            icon={GoogleLogo}
-            onClick={() => {
-              void signIn("google", {
-                callbackUrl: signUpCallbackUrl,
-              });
-            }}
-          />
+          {
+            <SignInButton
+              label="Sign up with Google"
+              icon={GoogleLogo}
+              onClick={handleSignUpClick}
+            />
+          }
         </div>
         <div className="flex flex-col gap-3 pb-20">
-          <P>
-            <Strong>Dust</Strong> is a platform giving you access to{" "}
-            <Strong>the best AI assistants</Strong>.
-            <br />
-            It's easy to&nbsp;use and it's a&nbsp;great place for teams
-            to&nbsp;collaborate.
-          </P>
-          <P>
-            Learn more about Dust on{" "}
-            <A href="https://dust.tt/" target="_blank" variant="secondary">
-              our homepage
+          <p>
```

Comment:
What the difference between `<P>` and `<p>` 🤔? Do we still need the `<A>`?

Reply:
As discussed, I used a next.js Link instead of <A> (I realized it doesn't change much because we don't benefit from next.js optimizations as the page opens in a new tab but it's good for consistency nevertheless)